group/add-member/schema.json

```diff
@@ -60,15 +60,15 @@
   "definitions": {
     "messageId": {
       "type": "string",
-      "pattern": "^%[a-zA-Z0-9\\/+]{42}[AEIMQUYcgkosw048]=.sha256$"
+      "pattern": "^ssb:message\/classic\/[a-zA-Z0-9_\\-]{43}[AEIMQUYcgkosw048]=$"
```

Comment:
I changed it to 43 because that's what it said over here

https://github.com/ssbc/ssb-uri2/blob/00151c68b6972dbfb60393aa710244a8a06b191a/src/index.ts#L270

but why is it different? And should I change the uris in these docs to match?

Reply:
Eeh, good question. I would assume it's correct in ssb-uri2 because if it would be wrong, then no SSB URIs would become links in Manyverse.

Reply:
but if this schema's wrong then things would error in tribes1 when messages are published and try to match this spec

Reply:
in uri2 the functions 43 is in is `getMessageSSBURIRegex` and `getFeedSSBURIRegex`, does anything actually use those regexes?

Reply:
i figured it out!

uri2 declares
43 chars + 1 =

private group spec declares
42 chars + 1 char from a limited subset (last bits are 0) + 1 =

so this spec is actually just more thorough. added a comment now and in the new tests i made the keys generated from proper crypto to not be random strings